.changeset/perfect-feet-approve.md

```diff
@@ -0,0 +1,6 @@
+---
+"@fuel-ts/account": minor
+"@fuel-ts/errors": minor
```

Comment:
If the API is the same, and we are just adding new validations, shouldn't this be a patch?